Packs/Akamai_SIEM/Integrations/Akamai_SIEM/README.md

```diff
@@ -269,4 +269,10 @@ The proposed solution in that case is to use the two parameters **Fetch limit**
 **Fetch limit** is the number of total events we want to retrieve each fetch interval.
 **Akamai Page size** is the number of events we want to retrieve each request. Note that the suggested maximum for **Akamai Page size** is 200k.
 Meaning that an interval may execute multiple requests and therefore you should configure **Akamai Page size** < **Fetch limit**
-You should work to find the balance between them in a way that both the command, and the request won't get any timeout.
\ No newline at end of file
+You should work to find the balance between them in a way that both the command, and the request won't get any timeout.
+
+### Known limitations
+
+## Each config ID should be configured on one instance at most:
+Due to limitations from Akamai side, each config ID can be configured on one instance at most on the same machine or on different machines (i.e, the same config ID can't be configured both on dev and prod tenants).
```

Comment:
Due to limitations from Akamai, the config ID can only be configured on one instance on the same machine or on different machines (i.e. the same config ID can't be configured both on dev and prod tenants).